sklearn/metrics/_ranking.py

```diff
@@ -742,7 +751,7 @@ def _binary_clf_curve(y_true, y_score, pos_label=None, sample_weight=None):
 
 
 @_deprecate_positional_args
-def precision_recall_curve(y_true, probas_pred, *, pos_label=None,
+def precision_recall_curve(y_true, y_score, *, pos_label=None,
```

Comment:
I agree `y_score` might have been better but we can't easily do this as it would break backward compatibility for people using key word arguments ( `precision_recall_curve(y_true=.., probas_pred=...)`.

If we do rename, we would probably need to go through a deprecation warning cycle. i.e. 
 - change `probas_pred` -> `y_score` or `y_pred` and make it an optional parameter.
 - add probas_pred as a key word argument.
 - raise a DeprecationWarning if probas_pred  is used, and make sure it's never used at the same time with y_pred

Let's wait for a second opinion, I'm not sure if it's worth to do it..

Reply:
You are right. I did not think about people using keyword arguments for positional ones...

Let's wait for some other opinion to decide whether revert this or start a deprecation warning cycle.

Reply:
If we do not rename, this PR might be much easier to get merged.

Reply:
I have reverted the renaming.